frontend/packages/kubevirt-plugin/integration-tests/tests/utils/consts.ts

```diff
@@ -56,12 +56,15 @@ Object.freeze(TABS);
 
 // Tab names
 export const VMACTIONS = {
```

Comment:
can we use the enum instead?

Reply:
No, I'm using these string in `data-test-id` selectors:
https://github.com/openshift/console/blob/b2d5d1daf8f830a0205c8ec905e8e12b2e20d4de/frontend/packages/kubevirt-plugin/integration-tests/views/vm.actions.view.ts#L32

Reply:
string based enum would work the same

Reply:
Ah ok, I though it maps to numbers. I'll take a look at it then